### ISSUE TITLE
Ammo not saving after gun put away and pulled back out 

### ISSUE THREAD

**Issue by Epicgamerjames** [qbcore-fivem/qb-weapons#193]
i pull my gun out and reload it and then place the gun back in my pocket and pull  it out again bullets gone and i have to load the gun again.
no f8 errors

**Comment by Sanriku-Gaming** [comment]
Is this after the latest update?
https://github.com/qbcore-framework/qb-weapons/commit/d87e3dca2a8168e65c6d2b897e514817af78c47e

**Comment by Epicgamerjames** [comment]
yes sir i can now reload my gun but the bullets disappear once i put it away and get it back out again

**Comment by ericb160** [comment]
having the same issue

**Comment by ericb160** [comment]
same here

**Comment by Sanriku-Gaming** [comment]
Sounds like you are using the latest qb-weapons with a different inventory. Unless you are using the latest qb-inventory, it's going to have issues.

**Comment by Elie4real** [comment]
> Sounds like you are using the latest qb-weapons with a different inventory. Unless you are using the latest qb-inventory, it's going to have issues.

so you cant use Qb-weapons with PS-inventory because that is my issue for not having my ammo save on the weapon when putting it away , and then pulling it back out ?

**Comment by autfoxx** [comment]
> Sounds like you are using the latest qb-weapons with a different inventory. Unless you are using the latest qb-inventory, it's going to have issues.
It's still throwing the same issue for me, not too sure what's causing it
